pkg/metrics/resources.go

```diff
@@ -1,6 +1,7 @@
 package metrics
 
 // Resource represents the resource that we want to time.
+//
```

Comment:
I think this is a go1.19 thing. Since we are updating the file anyway, might as well keep it, but I have also had the displeasure of dealing with this when trying to style this repo

Reply:
Yeah, I saw that and reverted to go 1.18